mailbox/api/src/main/java/org/apache/james/mailbox/events/MailboxEvents.java

```diff
@@ -415,10 +415,14 @@ public ImmutableSet<MessageId> getMessageIds() {
 
     class Expunged extends MetaDataHoldingEvent {
         private final Map<MessageUid, MessageMetaData> expunged;
+        private final Optional<MailboxId> movedToMailboxId;
 
-        public Expunged(MailboxSession.SessionId sessionId, Username username, MailboxPath path, MailboxId mailboxId, Map<MessageUid, MessageMetaData> uids, EventId eventId) {
+        public Expunged(MailboxSession.SessionId sessionId, Username username,
+                        MailboxPath path, MailboxId mailboxId, Map<MessageUid, MessageMetaData> uids,
+                        EventId eventId, Optional<MailboxId> movedToMailboxId) {
             super(sessionId, username, path, mailboxId, eventId);
             this.expunged = ImmutableMap.copyOf(uids);
+            this.movedToMailboxId = movedToMailboxId;
```

Comment:
I don't think this field is useful as you only use the one from Added event when `isMoved` and we skip the Expunged one :)

Reply:
I think the value of MailboxId is better than the boolean value, (maybe in another story?)
The code cost for MailboxId field or Boolean field is same.

Reply:
Ouh my bad I thought there was still a variable for the boolean, didnt realize it's just this one now. Alright :)